src/Mirza.Web/Pages/Index.cshtml

```diff
@@ -30,12 +30,12 @@
                         </div>
                         <div class="form-group col-md-4">
                             <label asp-for="Input.Start">شروع از</label>
-                            <input class="form-control" asp-for="Input.Start">
+                            <input id="input-start" class="form-control" name="Input.Start" type="text" value="@Model.Input.Start">
```

Comment:
I'd suggest using the tag-helper syntax:

`<input id="input-start" class="form-control" type="text" asp-for="Input.Start">`

Reply:
Fixed